libfreerdp/codec/h264.c

```diff
@@ -71,7 +71,7 @@ struct _H264_CONTEXT_OPENH264
 {
 	ISVCDecoder* pDecoder;
 	ISVCEncoder* pEncoder;
-	SEncParamBase EncParamBase;
+	SEncParamExt EncParamExt;
```

Comment:
@hardening SEncParamExt is always available. It is defined in public header codec_app_def.h side-by-side along with SEncParamBase and there is no conditional #if whatsoever. So is the InitializeExt API.